hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -354,7 +363,7 @@ private XceiverClientReply sendCommandWithRetry(
         responseProto = null;
       } catch (ExecutionException e) {
         LOG.debug("Failed to execute command {} on datanode {}",
-            request, dn.getUuid(), e);
```

Comment:
Full DatanodeDetails.toString() include a lot of information and can hurt the perf on critical path. Consider using either ip or uuid instead.

Reply:
I hope to using ip and uuid like ip/uuid, do you think it is ok for you?
Because, i want to get the ip so that i can clearly know the exception related to which datanode, but in local mode, all of ips of datanode are the same, for this situation, an addition uuid can help me to distinguish.

Reply:
I think ip address should be sufficient for production environment. The local mode is for mini clsuter test only.